modules/apps/users-admin/users-admin-test/src/testIntegration/java/com/liferay/users/admin/change/tracking/test/UsersAdminPortletTest.java

```diff
@@ -0,0 +1,1009 @@
+import com.liferay.portal.kernel.test.rule.AggregateTestRule;
+import com.liferay.portal.kernel.test.util.OrganizationTestUtil;
+import com.liferay.portal.kernel.test.util.RandomTestUtil;
+import com.liferay.portal.kernel.test.util.RoleTestUtil;
+import com.liferay.portal.kernel.test.util.ServiceContextTestUtil;
+import com.liferay.portal.kernel.test.util.TestPropsValues;
+import com.liferay.portal.kernel.test.util.UserGroupTestUtil;
+import com.liferay.portal.kernel.test.util.UserTestUtil;
+import com.liferay.portal.kernel.theme.ThemeDisplay;
+import com.liferay.portal.kernel.util.Constants;
+import com.liferay.portal.kernel.util.HashMapBuilder;
+import com.liferay.portal.kernel.util.StringUtil;
+import com.liferay.portal.kernel.util.WebKeys;
+import com.liferay.portal.test.rule.Inject;
+import com.liferay.portal.test.rule.LiferayIntegrationTestRule;
+import com.liferay.portal.test.rule.PermissionCheckerMethodTestRule;
+import com.liferay.portlet.ActionRequestFactory;
+import com.liferay.users.admin.constants.UsersAdminPortletKeys;
+
+import java.util.Calendar;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+
+import javax.portlet.ActionRequest;
+import javax.portlet.Event;
+import javax.portlet.Portlet;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.ClassRule;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+/**
+ * @author Brooke Dalton
+ */
+@RunWith(Arquillian.class)
+public class UsersAdminPortletTest {
+
+	@ClassRule
+	@Rule
+	public static final AggregateTestRule aggregateTestRule =
+		new AggregateTestRule(
+			new LiferayIntegrationTestRule(),
+			PermissionCheckerMethodTestRule.INSTANCE);
+
+	@Before
+	public void setUp() throws Exception {
+		_ctCollection = _ctCollectionLocalService.addCTCollection(
+			null, TestPropsValues.getCompanyId(), TestPropsValues.getUserId(),
+			0, "P1", null);
+
+		_layout = _layoutLocalService.getLayout(TestPropsValues.getPlid());
+	}
+
+	@Test
+	public void testNoCTEntryForAddedEditUserMVCAction() throws Exception {
+		String addUserScreenName = RandomTestUtil.randomString();
+
+		try (SafeCloseable safeCloseable =
+				CTCollectionThreadLocal.setCTCollectionIdWithSafeCloseable(
+					_ctCollection.getCtCollectionId())) {
+
+			_portlet.processAction(
+				_getActionRequest(
+					"/users_admin/edit_user",
+					HashMapBuilder.put(
+						Constants.CMD, Constants.ADD
+					).put(
+						"birthdayDay", String.valueOf(1)
+					).put(
+						"birthdayMonth", String.valueOf(Calendar.JANUARY)
+					).put(
+						"birthdayYear", String.valueOf(1970)
+					).put(
+						"emailAddress",
+						RandomTestUtil.randomString() + "@liferay.com"
+					).put(
+						"firstName", RandomTestUtil.randomString()
+					).put(
+						"lastName", RandomTestUtil.randomString()
+					).put(
+						"screenName", addUserScreenName
+					).build()),
+				new CustomMockLiferayPortletActionResponse());
```

Comment:
Same with this line

Reply:
Hey Noor! Would it make sense to also include the 
```
 try (SafeCloseable safeCloseable =
				CTCollectionThreadLocal.setCTCollectionIdWithSafeCloseable(
					_ctCollection.getCtCollectionId())
```
  in the "_processActionRequest" ?

Reply:
Yeah, definitely! 
I would pass in the ctCollectionId into the method as a parameter though. That increases the method's usability for other ctCollections